README.md

```diff
@@ -35,35 +36,36 @@ It is intended that connecting the bits together into an engine be done at the l
 
 ### Advantages of Libdogecoin
 
-* No dependencies in case no p2p network client is required (only dependency is [libsecp256k1](https://github.com/bitcoin-core/secp256k1) added as git subtree)
-* The only dependency for the p2p network client is [libevent](https://github.com/libevent/libevent) (very portable)
-* optimized for MCU and low mem environments
-* ~full test coverage
-* mem leak free (valgrind check during CI)
+- No dependencies in case no p2p network client is required (only dependency is [libsecp256k1](https://github.com/bitcoin-core/secp256k1) added as git subtree)
```

Comment:
Is it correct right now that our only deps are libevent and secp?  Can we build without unistd now; has it been internalized? Not sure.  (And libevent -should- self build as well as secp.)  Just want to make sure that this is a correct statement when, say, I go build on some weird non-debian based distro or something. (Which I should try.)

Reply:
afaik you have to pass -DWITH_UNISTRING=0 or --disable-unistring to disable so that statement is a tad bit outdated. i'll push an update to reflect libunistring is a dependency now but am in agreement with goal to have 0 deps.